core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -125,7 +125,9 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     RoadsConnectAcrossRivers("Roads connect tiles across rivers", UniqueTarget.Global),
     RoadMaintenance("[relativeAmount]% maintenance on road & railroads", UniqueTarget.Global),
     NoImprovementMaintenanceInSpecificTiles("No Maintenance costs for improvements in [tileFilter] tiles", UniqueTarget.Global),
+    @Deprecated("As of",ReplaceWith("[relativeAmount]% [All] improvement construction time"))
     TileImprovementTime("[relativeAmount]% tile improvement construction time", UniqueTarget.Global, UniqueTarget.Unit),
+    SpecificImprovementTime("[relativeAmount]% [improvementFilter] improvement construction time", UniqueTarget.Global, UniqueTarget.Unit),
```

Comment:
Is this necessary? Sounds to me like maybe what we actually need is an improvementFilter conditional

If we are going this route though, let's reword this to "[relativeAmount]% construction time for [improvementFilter] improvements"

Reply:
Also lower-case "all" should be an option, because the current uniques make me wince when I see an upper-case All in the middle of a sentence...

Reply:
> Sounds to me like maybe what we actually need is an improvementFilter conditional

Maybe, but unlike, say, a cit6Filter conditional, I'm not sure how often that'll come up. Off the top of my head, this is probably one of the only uniques that don't already support improvements as a tileFilter and that'll make sense for a conditional to apply to. The only other immediate alternative I can think of is a maintenance reducer for improvements

I can go ahead and reword it though

Reply:
> Also lower-case "all" should be an option

Probably something more to add to the list of available  improvement filters (and other filters for that matter), not entirely in scope here